src/main/java/Cell.java

```diff
@@ -24,4 +24,9 @@ public void putStone(@NotNull Stone stone) {
         this.occupied = true;
     }
 
+    public void clear() {
+        this.stone = null;
+        this.occupied = false;
```

Comment:
Occupied field may be unnecessary, just perform a check on this.stone != null

Reply:
Yes, I will remove it